dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -110,6 +123,10 @@ public int getPosition() {
         return javaType;
     }
 
+    protected TypeTransformerFactory getDbfitToJdbcXformFactory() {
```

Comment:
Some wondering about `xform` vs `transform` (xform might mean some different things, ma it be misleading?).